### PR TITLE
Changes for US Privacy String

### PR DESCRIPTION
**Pull request by rajat-kasliwal-inmobi**

### CODE REVIEW

File: InMobi/InMobiAdapter/ALInMobiMediationAdapter.m [hunk 1]
```diff
@@ -432,6 +436,15 @@ - (BOOL)showFullscreenAd:(IMInterstitial *)interstitial forParameters:(id<MAAdap
     }
 }
 
+- (void)setPrivcayCompliance:(id<MAAdapterParameters>)parameters {
```
Comment (anilbakhla) [inline InMobi/InMobiAdapter/ALInMobiMediationAdapter.m]: Correct the spelling

**Review by anilbakhla — CHANGES_REQUESTED** [review]

**Review by rishabdutta — APPROVED** [review]

File: InMobi/InMobiAdapter/ALInMobiMediationAdapter.m [hunk 2]
```diff
@@ -7,9 +7,9 @@
 //
 
 #import "ALInMobiMediationAdapter.h"
-@import InMobiSDK;
+#import <InMobiSDK/InMobiSDK.h>
```
Comment (anilbakhla) [inline InMobi/InMobiAdapter/ALInMobiMediationAdapter.m:10]: why there is change in header as MAX adapter is already in Swift

**Review by anilbakhla — CHANGES_REQUESTED** [review]

File: InMobi/InMobiAdapter/ALInMobiMediationAdapter.m (hunk 2 above)
Comment (rajat-kasliwal-inmobi) [reply InMobi/InMobiAdapter/ALInMobiMediationAdapter.m:10]: they have again done this change, that's why provided this reference link .
https://github.com/InMobi/AppLovin-MAX-SDK-iOS/pull/4/files

**Review by anilbakhla — APPROVED** [review]
LGTM

**Pull request merged**